src/palace_tools/constants.py

```diff
@@ -1,6 +1,6 @@
 # Some defaults
 DEFAULT_ASYNC_TIMEOUT = 30.0
-DEFAULT_AUTH_DOC_PATH_SUFFIX = "/authentication-document"
+DEFAULT_AUTH_DOC_PATH_SUFFIX = "authentication_document"
```

Comment:
This needed to be updated to be able to use the `OPDS Server Heuristic` args to `patron-bookshelf`.